client/components/jobs/JobDetail.tsx

```diff
@@ -0,0 +1,192 @@
+              augue id ex luctus vestibulum. Aenean sed urna est. Praesent
+              iaculis mauris a eros rutrum viverra.
+            </Text>
+          </Stack>
+
+          <Stack>
+            <Heading fontSize="lg">Requirements</Heading>
+            <UnorderedList>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+            </UnorderedList>
+          </Stack>
+
+          <Stack>
+            <Heading fontSize="lg">Responsibilities</Heading>
+            <UnorderedList>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+              <ListItem>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+              </ListItem>
+            </UnorderedList>
+          </Stack>
+        </Stack>
+      </Container>
+
+      <Stack
+        as="aside"
+        p={7}
+        w={{ base: '100%', md: '70%', lg: '365px' }}
+        spacing={6}
+        borderColor="gray.800"
+        borderWidth={2}
+        rounded="md"
+        divider={
+          <StackDivider borderColor="gray.400" borderWidth={2} rounded="md" />
+        }
+      >
+        <Stack textAlign="center" spacing={3}>
+          <Stack spacing={1}>
+            <Heading as="h1">GitCoin</Heading>
+            <Link fontSize="sm">Visit Website</Link>
+          </Stack>
+          <ApplyButton display={{ sm: 'none', md: 'block' }}>
+            Apply For This Job
+          </ApplyButton>
+        </Stack>
+
+        <Stack spacing={4}>
+          <Detail title="Job Type">Full-Time</Detail>
+          <Detail title="Location">California</Detail>
+          <Detail title="Position">Marketing</Detail>
+          <Detail title="Offered Salary">10,000—15,000 or 2-3 ETH</Detail>
+          <Detail title="Equity">upto +5%</Detail>
+        </Stack>
+      </Stack>
+      <Stack
+        display={{ sm: 'flex', md: 'none' }}
+        w="100vw"
+        position="sticky"
+        bottom={0}
+        bg="#f7fafc"
+        left={0}
+        py="6"
+        borderTopColor="gray.800"
+        borderTopWidth={2}
+        align="center"
+      >
+        <ApplyButton w="70%">Apply For This Job</ApplyButton>
+      </Stack>
+    </Stack>
+  );
+}
+
+const Detail = ({
```

Comment:
What is this for?

Reply:
the Detail component is built by with-heart to give detail about the job in the ```job card``` by simply using the detail component you can write the title and its below text simply.
![image](https://user-images.githubusercontent.com/57826091/142806607-b9e2149a-1d13-4bd2-87fe-3108c52b22de.png)